### ISSUE TITLE
Benefit over sidechains?

### ISSUE THREAD

**Issue by iancoleman** [Frogolocalypse/Bitcoin_non_coerced_hardfork#4]
I think there's a natural comparison to be made with sidechains, especially with the mechanism to transfer coins to hardforks.

It might be beneficial to address the differences and benefits of this proposal compared to sidechains.

https://gendal.me/2014/10/26/a-simple-explanation-of-bitcoin-sidechains/

**Comment by Frogolocalypse** [comment]
Yes, that has been pointed out already.  I highlighted that there is a fundamental difference in that there is a mechanism for aligning the coin release schedules between the two chains, but this needs to be better explained.

I will develop a graphic that better explains the concept.
